docs/UsersGuide/source/Components.rst

```diff
@@ -0,0 +1,81 @@
+.. _Components:
+
+============================
+SRW Application Components
+============================
+
+The SRW Application assembles a variety of components, including:
+
+* Pre-processor Utilities & Initial Conditions
+* UFS Weather Forecast Model
+* Unified Post-Processor
+* Visualization Examples
+* Build System and Workflow
+
+These components are documented within this User's Guide and supported through a `community forum <https://forums.ufscommunity.org/>`_. 
+
+.. _Utils:
+
+Pre-processor Utilities and Initial Conditions
+==============================================
+
+The SRW Application includes a number of pre-processing utilities that initialize and prepare the model. Since the SRW App provides forecast predictions over a limited area (rather than globally), it is necessary to first generate a regional grid (``regional_esg_grid/make_hgrid``) along with :term:`orography` (``orog``) and surface climatology (``sfc_climo_gen``) files on that grid. Grids include a strip, or "halo," of six cells that surround the regional grid and feed in lateral boundary condition data. Since different grid and orography files require different numbers of halo cells, additional utilities handle the correct number of halo shave points and topography filtering. The pre-processing software ``chgres_cube`` is used to convert the raw external model data into initial and lateral boundary condition files in netCDF format. These are needed as input to the FV3-LAM. Additional information about the UFS pre-processor utilities can be found in the `UFS_UTILS User’s Guide <https://noaa-emcufs-utils.readthedocs.io/en/ufs-v2.0.0/>`_.
```

Comment:
I am not sure on this so tagging @JeffBeck-NOAA 
Should it say "...correct number of halo shave points and..." or "...correct number of halo points and..."

Reply:
Also - is the UFS_UTILS User's Guide being updated to v3.0.0 for this SRW release? This should be coordinated to make sure it is linked here as appropriate.

Reply:
I might say something like, "additional utilities handle shaving the number of halo points (based on downstream workflow component requirements) and topography filtering".